examples/plot_mne_example.py

```diff
@@ -0,0 +1,223 @@
+"""
+Using fooof with MNE
+=================
+
+This examples illustrates how to use fooof with `MNE
+<https://mne-tools.github.io/>`_ and create topographical plots
+
+This tutorial does require that you have MNE installed. If you don't already have
+MNE, you can follow instructions to get it `here
+<https://mne-tools.github.io/stable/getting_started.html>`_.
+"""
+
+###################################################################################################
+
+
+#General imports
+import numpy as np
+import pandas as pd
+from matplotlib import cm, colors, colorbar
+import matplotlib.pyplot as plt
+
+# Import MNE, as well as the MNE sample dataset
+import mne
+from mne import io
+from mne.datasets import sample
+from mne.viz import plot_topomap
+
+# FOOOF imports
+from fooof import FOOOF, FOOOFGroup
+from fooof.analysis import *
+
+
+###################################################################################################
+# Load & Check MNE Data
+# ---------------------
+#
+# First, we will load the example dataset from MNE, and have a quick look at the data.
+#
+# The MNE sample dataset is a combined MEG/EEG recording with an audiovisual task, as
+# described `here <https://martinos.org/mne/stable/manual/sample_dataset.html>`_.
+#
+# For the current example, we are going to sub-select only the EEG data,
+# and analyze it as continuous (non-epoched) data.
+#
+
+###################################################################################################
+
+# Get the data path for the MNE example data
+raw_fname = sample.data_path() + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+event_fname = sample.data_path() + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
+
+# Load the file of example MNE data
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+
+###################################################################################################
+
+# Select EEG channels from the dataset
+raw = raw.pick_types(meg=False, eeg=True, eog=False, exclude='bads')
+
+###################################################################################################
+
+# Explicitly adding a default EEG average reference required for source localization
+raw.set_eeg_reference()
+
+###################################################################################################
+
+# Explore the data
+raw.info()
```

Comment:
This line gives me a Syntax Error (Info object is not callable). Info here is a dictionary (or similar), and shouldn't be called as a function (at least, I think that's what's going on). 

I think we probably don't need to print out the info object anyways.